package.json

```diff
@@ -40,7 +42,7 @@
         "release:version:auto": "yarn release:version --conventional-commits --no-changelog && yarn build:clean",
         "release:publish": "yarn test:clean && lerna publish from-package",
         "test": "jest --coverage --runInBand",
-        "test:ci": "yarn lint:all && yarn test --ci --reporters=default --reporters=jest-junit && yarn test:wdio",
+        "test:ci": "yarn lint:all:except:depgraph && yarn test --ci --reporters=default --reporters=jest-junit && yarn test:wdio",
```

Comment:
'package-dependency-graph' has an issue generating dep graph on Node 18.  Entered an [issue](https://github.com/plantain-00/package-dependency-graph/issues/14) item on the package.

For now, bypassing the dep graph generation check as part of CI processes, will see for a fix from package owners or look for an alternative as how it goes. [W-13534678](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001SxSZ9YAN/view) to track.